package.json

```diff
@@ -53,6 +53,7 @@
     "istanbul-reports": "^3.1.3",
     "jest": "workspace:^",
     "jest-changed-files": "workspace:^",
+    "jest-diff": "workspace:^",
```

Comment:
I had in mind that it should be added to `packages/expect/package.json`. `jest-diff` is dev dependency of `expect` package. Also remember to run `yarn` and to commit the lock file as well.

Reply:
[`2a07104` (#13362)](https://github.com/facebook/jest/pull/13362/commits/2a071046c47bc1ef221c00111fc397bcf6f2d475)